### DISCUSSION TITLE
How to "unlock" multiple profiles, all of them in"planned" state?

### DISCUSSION THREAD

**Discussion by johanneskastl** [floccusaddon/floccus#1785] [Q&A] [answered]
I have two profiles, syncing different folders of my Firefox profile to Nextcloud Bookmarks.

Sometimes both of them seem to end up in "planned" state ("Geplant" in German), apparently each waiting on the other to finish. This seems to only get solved once I quit Firefox and restart it.

Is there any other way to unlock this?

Kind Regards,
Johannes

**Comment by johanneskastl** [comment]
OK, now even restarting Firefox does not unlock this situation...

**Reply by marcelklehr** [reply, marked as answer]
In floccus v5.4.0 you can now force sync with a button that is displayed along with the explanation of the scheduled state.

**Reply by johanneskastl** [reply]
Huh, I did not see that button as clickable. But I'll try, thanks for the hint!

**Reply by marcelklehr** [reply]
You are likely not on v5.4.0 yet
